test_scripts/Smoke/Resumption/commonStepsResumption.lua

```diff
@@ -0,0 +1,71 @@
+local commonStepsResumption = {}
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+
+
+local function CheckTimeBoundaries(time)
+  local timeout = tonumber(commonFunctions:read_parameter_from_smart_device_link_ini("ApplicationResumingTimeout"))
+  return function (exp, _)
+    if exp.occurences == 2 then
+      local time2 = timestamp()
+      local time_to_resumption = time2 - time
+      if time_to_resumption >= timeout and time_to_resumption < (timeout + 500) then
```

Comment:
@istoimenova According to [HMILevel Resumption]: Resume HMILevel of an app after "resume timeout" expires : SDL must resume HMILevel of an app that satisfies conditions of HMILevel resumption - rignt after "resume timeout" value from .ini file ("ApplicationResumingTimeout") is expired.